phing/tasks/setup.xml

```diff
@@ -248,13 +248,17 @@
           <param>drush</param>
         </drush>
         <!-- Revert all features. -->
-        <drush command="fra" assume="yes" alias="${drush.alias}">
-          <option name="bundle">${cm.features.bundle}</option>
-        </drush>
+        <foreach list="${cm.features.bundle}" param="bundle" target="setup:update:features-import" />
```

Comment:
And to be clear I think 'bundles' is clearer semantically but again I wanted to make sure this didn't break anyone's build.